Week-3/CSS-ADVANCED/q3.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Playing cards</title>
+    <style>
+        .table {
+        width: 500px;
+        height: 300px;
+        background: linear-gradient(to bottom, forestgreen, darkgreen);
+
+      }
+
+      .cardBlack {
+        color: black;
+      }
+
+      .cardRed {
+        color: red;
+      }
+
+      .card {
```

Comment:
1. Use relative units like em or rem for font sizes, widths, and heights to improve responsiveness.
```
.card {
  width: 6.25em; /* instead of 100px */
  height: 10em; /* instead of 160px */
  /* ... other styles ... */
}
```

2. Use a container for the cards to manage their layout more efficiently.

```
.card-container {
  position: relative;
  width: 15em; /* Based on the card size */
  height: 10em;
}

```